homeassistant/components/islamic_prayer_times/sensor.py

```diff
@@ -38,8 +38,8 @@
         translation_key="maghrib",
     ),
     SensorEntityDescription(
-        key="Isha",
-        translation_key="isha",
+        key="Ishaa",
+        translation_key="ishaa",
```

Comment:
the key cannot change without changing the corresponding key in `strings.json`
the user facing string is on `strings.json`.